src/main/resources/assets/alexscaves/lang/zh_cn.json

```diff
@@ -264,305 +264,306 @@
   "block.alexscaves.thornwood_door": "荆棘木门",
   "block.alexscaves.thornwood_button": "荆棘木按钮",
   "block.alexscaves.thornwood_fence_gate": "荆棘木栅栏门",
-  "block.alexscaves.thornwood_sapling": "荆棘幼苗",
-  "block.alexscaves.potted_thornwood_sapling": "荆棘幼苗盆栽",
+  "block.alexscaves.thornwood_sapling": "荆棘木树苗",
+  "block.alexscaves.potted_thornwood_sapling": "荆棘木树苗花盆",
   "block.alexscaves.moth_ball": "驱蛾球",
-  "block.alexscaves.beholder": "窥探器",
-  "item.alexscaves.advancement_tab_icon": "Alex 的洞穴图标",
-  "item.alexscaves.cave_tablet": "洞穴石板",
-  "item.alexscaves.cave_codex": "洞穴手稿",
-  "item.alexscaves.cave_codex.add": "在《洞穴概论》里添加 %s：%s",
-  "item.alexscaves.cave_codex.end": "该生物群系的知识已尽数掌握。",
-  "item.alexscaves.cave_book": "洞穴概论",
-  "item.alexscaves.cave_book.desc": "由洞穴专家 Alexander Caverns 教授所著",
-  "item.alexscaves.cave_book.general": "综合介绍",
-  "item.alexscaves.cave_book.resources": "自然资源",
-  "item.alexscaves.cave_book.mobs": "栖息物种",
-  "item.alexscaves.cave_book.utilities": "实用物品",
+  "block.alexscaves.beholder": "远瞻者",
+  "item.alexscaves.advancement_tab_icon": "Alex's Caves Icon",
+  "item.alexscaves.cave_tablet": "洞穴石碑",
+  "item.alexscaves.cave_codex": "洞穴手抄本",
+  "item.alexscaves.cave_codex.add": "将 %s: %s 添加到洞穴宝典",
+  "item.alexscaves.cave_codex.end": "你已经收录了这个生物群系的所有知识",
+  "item.alexscaves.cave_book": "洞穴宝典",
+  "item.alexscaves.cave_book.desc": "Alexander Caverns博士 著",
+  "item.alexscaves.cave_book.general": "基本信息",
+  "item.alexscaves.cave_book.resources": "资源",
+  "item.alexscaves.cave_book.mobs": "生物",
+  "item.alexscaves.cave_book.utilities": "实用知识",

```

Comment:
**Inappropriate translation**. New text means "Useful knowledges", which doesn't match the meaning of "**_utility_**".